load-testing.md

```diff
@@ -0,0 +1,74 @@
+# Load testing the OCSP signing components.
+
+Here are instructions on how to realistically load test the OCSP signing
+components of Boulder, excercising the pkcs11key, boulder-ca, and
+ocsp-updater components.
+
+Set up a SoftHSM instance running pkcs11-daemon on some remote host with more
+CPUs than your local machine. Easiest way to do this is to clone the Boulder
+repo, and on the remote machine run:
+
+    remote-machine$ docker-compose run --service-ports bhsm
+
+Check that the port is open:
+
+    local-machine$ nc -zv remote-machine 5657
+    Connection to remote-machine 5657 port [tcp/*] succeeded!
+
+Initialize the necessary tokens:
+
+    local-machine$ alias drun="docker-compose run -e PKCS11_PROXY_SOCKET=tcp://remote-machine:5657 -e FAKE_DNS=172.17.0.1 --service-ports"
+    local-machine$ drun --entrypoint "softhsm --module /usr/local/lib/libpkcs11-proxy.so --init-token --pin 5678 --so-pin 1234 --slot 0 --label intermediate" boulder
+    local-machine$ drun --entrypoint "softhsm --module /usr/local/lib/libpkcs11-proxy.so --init-token --pin 5678 --so-pin 1234 --slot 1 --label root" boulder
+
+Run the pkcs11key benchmark to check raw signing speed at various settings for
+GOMAXPROCS and SESSIONS:
+
+    local-machine$ drun -e GOMAXPROCS=4 -e SESSIONS=4 -e MODULE=/usr/local/lib/libpkcs11-proxy.so --entrypoint /go/src/github.com/letsencrypt/pkcs11key/test.sh boulder
+
+Run a local Boulder instance:
+
+    local-machine$ drun boulder ./start.py
+
+Issue a bunch of certificates with test.js, ideally more than a hundred. Note:
+you may already have more than a hundred certificates already issued in your
+local database. If so, no need to do more.
+
+Using a MySQL client, artificially make all the OCSP responses go stale:
+
+    local-machine$ mysql -h 172.17.0.3 -u root -D boulder_sa_integration --silent
+    MariaDB [boulder_sa_integration]> update certificateStatus set ocspLastUpdated = DATE_SUB(NOW(), INTERVAL 9 DAY);
```

Comment:
Why `INTERVAL 9 DAY`? Is this related to an ocsp-updater configuration setting, if so which?